package.json

```diff
@@ -25,7 +25,7 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-register": "^6.9.0",
     "chai": "^4.0.2",
-    "graphql": "^14.0.2",
+    "graphql": "^14.3.0",
```

Comment:
`stripIgnoredCharacters ` was added in this version and is not available in older versions.